src/System.Reflection.MetadataLoadContext/src/FromCoreRt/System/Reflection/Runtime/BindingFlagSupport/MethodPolicies.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using RuntimeTypeInfo = System.Reflection.TypeLoading.RoType;
+
+namespace System.Reflection.Runtime.BindingFlagSupport
+{
+    //==========================================================================================================================
+    // Policies for methods.
+    //==========================================================================================================================
+    internal sealed class MethodPolicies : MemberPolicies<MethodInfo>
+    {
+        public sealed override IEnumerable<MethodInfo> GetDeclaredMembers(TypeInfo typeInfo)
+        {
+            return typeInfo.DeclaredMethods;
+        }
+
+        public sealed override IEnumerable<MethodInfo> CoreGetDeclaredMembers(RuntimeTypeInfo type, NameFilter filter, RuntimeTypeInfo reflectedType)
+        {
+            return type.GetMethodsCore(filter, reflectedType);
+        }
+
+        public sealed override bool AlwaysTreatAsDeclaredOnly => false;
+
+        public sealed override void GetMemberAttributes(MethodInfo member, out MethodAttributes visibility, out bool isStatic, out bool isVirtual, out bool isNewSlot)
+        {
+            MethodAttributes methodAttributes = member.Attributes;
+            visibility = methodAttributes & MethodAttributes.MemberAccessMask;
+            isStatic = (0 != (methodAttributes & MethodAttributes.Static));
+            isVirtual = (0 != (methodAttributes & MethodAttributes.Virtual));
+            isNewSlot = (0 != (methodAttributes & MethodAttributes.NewSlot));
+        }
+
+        public sealed override bool ImplicitlyOverrides(MethodInfo baseMember, MethodInfo derivedMember)
+        {
+            // TODO (https://github.com/dotnet/corert/issues/1896) Comparing signatures is lame. The runtime and/or toolchain should have a way of sharing this info.
```

Comment:
This issue is closed and has little info, you could remove the TODO

Reply:
Done